source/common/modules/markdown-editor/parser/frontmatter-parser.ts

```diff
@@ -49,6 +50,9 @@ export const frontmatterParser: BlockParser = {
       // The parser has collected the full rest of the document. This means
       // the frontmatter never stopped. In order to maintain readability, we
       // simply abort parsing.
+      const node = ctx.elt('YAMLFrontmatterStart', 0, 3)
+      ctx.nextLine()
+      ctx.addElement(node)
```

Comment:
Please remove this. There is a reason an unclosed frontmatter is not parsed as a frontmatter. This will break compatibility with Pandoc and break expectations.